src/metrics/prometheus_quantile_summary.erl

```diff
@@ -45,33 +45,18 @@ request_size_bytes\{quantile=\"0.95\"\}
 ```
 
 ### Configuration
-The specs cannot have a key called `quantile`, as this key is reserved by the implementation.
-
-Passing `targets`, the quantile estimator can be fine-tuned, as in
-`quantile_estimator:f_targeted/1`. See `default_targets/0` for the default values.
-
-Passing `compress_limit` we can configure how often to run compressions into the quantile summary
-algorithm.
-
-### Notes
-
-The underlying algorithm implemented in `m:quantile_estimator` does not support mergeability of
-summaries, therefore a summary in this implementation is linear, that is, all updates happen on the
-same ets record. Because of this, race conditions can imply datapoint loss.
-Statistically, this might not be relevant, but such impact is not ensured.
-
-Because of this, support for quantile summaries is rather experimental
-and histograms are recommended instead.
+See `t:ddskerl_ets:opts/0` for configuration options. These need to be passed as a proplist.
```

Comment:
Doesn't this mean that a wrapper function for this would require importing `ddskerl` for Dialyzer to not complain? Would a better approach be to replicate the type with a different name in `prometheus.erl`?

Reply:
(unless this is not supposed to be exported interface, ofc)

Reply:
It is not an exported interface, only documentation. Specs require here to take proplists.